UIKitDz/UIKitDz/FifthViewController.swift

```diff
@@ -0,0 +1,141 @@
+        
+        let oneArrayLabel = [oneLabel, twoLabel, threeLabel, fourLabel, fiveLabel]
+  
+        titleView.frame = CGRect(x: 200, y: 50, width: 150, height: 50)
+        titleView.text = "Оплата"
+        titleView.textAlignment = .center
+        titleView.font = UIFont(name: "Helvetica", size: 22)
+        titleView.center.x = view.center.x
+        view.addSubview(titleView)
+        
+        paymentButton.frame = CGRect(x: 100, y: 750, width: 300, height: 45)
+        paymentButton.backgroundColor = .black
+        paymentButton.center.x = view.center.x
+        paymentButton.addTarget(self, action: #selector (goToSecondView(sender:)), for: .touchUpInside)
+        view.addSubview(paymentButton)
+        
+        paymentButton.layer.cornerRadius = 5
+        paymentButton.setImage(UIImage(systemName: "applelogo"), for: .normal)
+        paymentButton.tintColor = .white
+        paymentButton.setTitle("Pay", for: .normal)
+        
+        orderLabel.frame = CGRect(x: 50, y: 100, width: 200, height: 50)
+        orderLabel.text = "Ваш заказ:"
+        view.addSubview(orderLabel)
+        
+        oneArrayLabel[0].frame = CGRect(x: 50, y: 150, width: 300, height: 50)
+        oneArrayLabel[1].frame = CGRect(x: 50, y: 200, width: 300, height: 50)
+        oneArrayLabel[2].frame = CGRect(x: 50, y: 250, width: 300, height: 50)
+        oneArrayLabel[3].frame = CGRect(x: 50, y: 300, width: 300, height: 50)
+        oneArrayLabel[4].frame = CGRect(x: 50, y: 350, width: 300, height: 50)
+        view.addSubview(oneArrayLabel[0])
+        
+        cardSwitch.frame = CGRect(x: 300, y: 650, width: 100, height: 50)
+        cardSwitch.addTarget(self, action: #selector(choicePayment), for: .allEvents)
+        view.addSubview(cardSwitch)
+        
+        cashSwitch.frame = CGRect(x: 300, y: 700, width: 100, height: 50)
+        cashSwitch.isOn = true
+        cashSwitch.addTarget(self, action: #selector(choicePayment), for: .allEvents)
+        view.addSubview(cashSwitch)
+        
+        textCardLabel.frame = CGRect(x: 50, y: 640, width: 250, height: 50)
+        textCardLabel.text = "Оплата картой"
+        textCardLabel.font = UIFont(name: "Helvetica", size: 22)
+        view.addSubview(textCardLabel)
+        
+        textCashLabel.frame = CGRect(x: 50, y: 690, width: 250, height: 50)
+        textCashLabel.text = "Оплата наличными"
+        textCashLabel.font = UIFont(name: "Helvetica", size: 22)
+        view.addSubview(textCashLabel)
+        
+        outputMenu(arrayLabel: oneArrayLabel, arraySelect: selectedAdd)
+    }
+ //MARK: - Public metods
+    func outputMenu(arrayLabel: [UILabel], arraySelect: [String]) {
+        for (index, value) in arraySelect.enumerated() {
+            arrayLabel[index].text = "\(index+1) " + value
+            arrayLabel[index].font = UIFont(name: "Helvetica", size: 24)
+            view.addSubview(arrayLabel[index])
+        }
+    }
+  
+    @objc func  choicePayment(target: UISwitch) {
```

Comment:
```suggestion
    @objc private func  choicePayment(target: UISwitch) {
```
их тоже private помечаем, если не вызываются из вне